### DISCUSSION TITLE
Intent push to terminal

### DISCUSSION THREAD

**Discussion by gyroscopes** [dojo-engineering/feedback#10] [Ideas]
The Dojo API has the intent call which sets everything up ready for an online transaction.

Wouldn't it be great to be able to have a optional parameter for a card machine ID (physical machine).
When this parameter is supplied, assuming it is valid and registered to the account the card machine would await payment (amount details on card machine ready).

**Comment by RomanKondratyev** [comment]
@gyroscopes yes, it would!
However, today remote payments APIs and card machines APIs are not connected at all... We might go this direction to unify things, but this is something long term. Today, to work with a card machine, you need to use this API https://docs.connect.paymentsense.cloud/rest/getting-started

**Comment by RomanKondratyev** [comment]
@jackhopner jfyi

**Comment by RomanKondratyev** [comment]
@gyroscopes not sure if you are still interested, but the intent to terminal is working now.
